### PR TITLE
Add lightweight cache plugin to fastify ecosystem

### PR DESCRIPTION
**Pull request by denbon05**
#### Checklist

- [x] run `npm run test`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by denbon05** [comment]
@allevo Hi. 
I was trying to add a plugin to Fastify ecosystem, but without success.
Can I ask you for approving review?

**Comment by luisorbaiceta** [comment]
Please hold the workflow run. Link check action will fail until #4078 is merged

File: docs/Guides/Ecosystem.md [hunk 1]
```diff
@@ -406,6 +406,8 @@ section.
 - [`fastify-orientdb`](https://github.com/mahmed8003/fastify-orientdb) Fastify
   OrientDB connection plugin, with which you can share the OrientDB connection
   across every part of your server.
+- [`fastify-lcache`](https://github.com/denbon05/fastify-lcache)
```
Comment (climba03003) [inline docs/Guides/Ecosystem.md:409]: Please follow alphabetical order.

**Review by climba03003 — CHANGES_REQUESTED** [review]

**Review by mcollina — APPROVED** [review]
lgtm

**Review by climba03003 — APPROVED** [review]
LGTM.

**Comment by luisorbaiceta** [comment]
LGTM is an abbreviation of "looks good to me". The list follows alphabetical order

**Comment by luisorbaiceta** [comment]
Could you please address the alphabetical order? @denbon05

**Comment by denbon05** [comment]
@luisorbaiceta, thank you for explanation. As I see plugin is already merged, do I need make some changes in the plugin list now ot it can stay as is it?

**Comment by luisorbaiceta** [comment]
> do I need make some changes in the plugin list now

That would be great. You would need to create a new PR though

**Pull request merged**